api/Report.json

```diff
@@ -11,7 +11,15 @@
           "chrome_android": "mirror",
           "edge": "mirror",
           "firefox": {
-            "version_added": false
+            "version_added": "65",
+            "flags": [
+              {
+                "type": "preference",
+                "name": "dom.reporting.enabled",
+                "value_to_set": "true"
+              }
+            ],
+            "notes": "Worker support added in version 77"
```

Comment:
No point adding worker support record here yet because it only becomes relevant if FF releases without worker support.